content/docs/2.7/scalers/cpu.md

```diff
@@ -18,9 +18,9 @@ This specification describes the `cpu` trigger that scales based on cpu metrics.
 ```yaml
 triggers:
 - type: cpu
+  metricType: Utilization/ AverageValue
```

Comment:
The same info should be added to Memory scaler.

Reply:
good catch, thanks!